server/src/app/game_manager.py

```diff
@@ -0,0 +1,116 @@
+import string
+import random
+
+class GameManager:
+    def __init__(self, redis_client):
+        self.redis_client = redis_client
+        if not self.redis_client.exists("breach:games_count"):
+            self.redis_client.set("breach:games_count", 0)
+
+    def create_game(self):
+        new_game_id = int(self.redis_client.get("breach:games_count")) + 1
+        self.redis_client.set("breach:games_count", new_game_id)
+        board = self.generate_random_board(new_game_id)
+
+        # Create token for joining the game
+        token = self.generate_random_token()
+        self.redis_client.hset("breach:token", token, new_game_id)
+
+        self.redis_client.hset("breach:game_status", new_game_id, "started;waiting_player2")
```

Comment:
Can you please undo all turn related changes from this CL?

Reply:
changes done